lib/api/layout/functions.php

```diff
@@ -224,22 +224,39 @@ function _beans_get_layout_classes( array $args ) {
 function beans_get_layouts_for_options( $add_default = false ) {
 	$base    = BEANS_ADMIN_ASSETS_URL . 'images/layouts/';
 	$layouts = array(
-		'c' => $base . 'c.png',
+		'c' => array(
+			'src'                => $base . 'c.png',
+			'alt'                => 'Full-Width Content Layout',
```

Comment:
@christophherr Let's pass all of thee through `__( '', 'tm-beans' );` for internationalization.